Syntaxes/Diagnostics.sublime-syntax

```diff
@@ -42,16 +42,16 @@ contexts:
 
   expect-diag-type:
     # See: https://github.com/sublimehq/Packages/issues/1036
-    # We use old markup scopes too so that old color schemes can catch up.
+    # We use sublimelinter markup scopes too so that old color schemes can catch up.
     - include: pop-at-end
     - match: \berror\b
-      scope: markup.error.lsp markup.deleted.lsp
+      scope: markup.error.lsp sublimelinter.mark.error
       pop: true
     - match: \bwarning\b
-      scope: markup.warning.lsp markup.heading.lsp
```

Comment:
Can we have all three groups, with the sublimelinter scopes in between the markup classes? We'd have (future scopes without hacks) > (correct colors but not reliable) > (at least something is visible)